### ISSUE TITLE
Handle dark mode for Gravity Forms error summary & error state labels

### ISSUE THREAD

**Issue by joedolson** [wpa11yday/wpaccessibilityday#98]
The color contrast on the list of errors is fine, but should be adjusted to be more dark-mode friendly; the labels are not high enough contrast.

- [ ] Adjust error list colors
- [ ] Adjust label colors in dark mode

![image](https://github.com/wpa11yday/wpaccessibilityday/assets/1686451/66a65bc2-a823-41cc-a703-31b0f50d6ee8)
